RecoParticleFlow/PFClusterTools/test/ProducePFCalibrationObject.cc

```diff
@@ -10,21 +10,20 @@
 
 #include <iostream>
 #include <vector>
-#include "CondFormats/PhysicsToolsObjects/interface/PerformancePayloadFromTFormula.h"
+//#include "CondFormats/PhysicsToolsObjects/interface/PerformancePayloadFromTFormula.h"
```

Comment:
Please remove if not needed any more (but it is probably better to have it included here rather than in the header file)